src/representations/representation_manager_spherical_invariants.hh

```diff
@@ -482,8 +583,7 @@ namespace rascal {
                      ++cartesian_idx) {
                   size_t cartesian_offset_n{cartesian_idx * this->max_radial};
                   size_t cartesian_offset_n1n2{
-                      cartesian_idx *
-                      static_cast<size_t>(math::pow(this->max_radial, 2))};
+                      cartesian_idx * math::pow(this->max_radial, 2_z)};
```

Comment:
Ok, now I appreciate the brevity of the `math::pow` functions with this new `_z` prefix (though I still think it should be `_n` or `_u`)